src/style.css

```diff
@@ -0,0 +1,35 @@
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-image: url(./images/star.svg);
+  background-position: center center;
```

Comment:
```suggestion
  background-position: center;
```

Reply:
I have fixed all issues and attached the links.